code/modules/mob/dead/observer/observer.dm

```diff
@@ -657,6 +657,33 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 
 	forceMove(tree.entrance)
 
+/mob/dead/observer/verb/teleport_z_up()
+	set category = "Ghost"
+	set name = "Move up"
+	set desc = "Move up a z level"
+
+	if(!istype(usr, /mob/dead/observer))
+		to_chat(src, "<span style='color: red;'>Not when you're not dead!</span>")
```

Comment:
who tf else is going to have this verb this is basically istype(src but totally redundant